### ISSUE TITLE
Improve Keyboard Support

### ISSUE THREAD

**Issue by dennis531** [elan-ev/studip-opencast-plugin#1235]
**Is your feature request related to a problem? Please describe.**
The keyboard support in the plugin should be improved. It should at least be possible to play videos in the course.
